src/components/helpers/factory.ts

```diff
@@ -0,0 +1,19 @@
+import { addCssToElement } from './css'
+
+export const createSpan = (
+  className: string,
+  title: string,
+  styleName: string
```

Comment:
I don't understand, why you've created two different parameters for a class name. Anyway, this will be irrelevant if you will use the method from Peter's comment.